src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -95,10 +107,18 @@ public void satisfiesEqualsContract() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            new RqMultipart.Base(req),
-            Matchers.equalTo(new RqMultipart.Base(req))
-        );
+        final RqMultipart.Base reqbaseone = new RqMultipart.Base(req);
+        final RqMultipart.Base reqbasetwo = new RqMultipart.Base(req);
+        try {
+            MatcherAssert.assertThat(
+                reqbaseone,
+                Matchers.equalTo(reqbasetwo)
+            );
+        } finally {
+            req.body().close();
```

Comment:
@raphaelln why these changes are needed? Can you point me to specific sentence in PR description?